plugin.video.vtm.go/resources/lib/vtmgo/vtmgostream.py

```diff
@@ -0,0 +1,473 @@
+            )
+
+        for subtitle in subtitles:
+            output_file = temp_dir + '/' + subtitle.split('/')[-1].split('.')[0] + '.nl-NL.' + subtitle.split('.')[-1]
+            webvtt_content = requests.get(subtitle).text
+            webvtt_content = webvtt_timing_regex.sub(lambda match: self._delay_webvtt_timing(match, ad_breaks), webvtt_content)
+            with self._kodi.open_file(output_file, 'w') as webvtt_output:
+                webvtt_output.write(from_unicode(webvtt_content))
+            delayed_subtitles.append(output_file)
+        return delayed_subtitles
+
+    def _anvato_get_anvacks(self, access_key):
+        """ Get the anvacks from anvato. (not needed)
+        :type access_key: string
+        :rtype dict
+        """
+        url = 'https://access-prod.apis.anvato.net/anvacks/{key}'.format(key=access_key)
+        self._kodi.log('Getting anvacks from {url}', url=url)
+        response = self._session.get(url,
+                                     params={
+                                         'apikey': self._ANVATO_API_KEY,
+                                     },
+                                     headers={
+                                         'X-Anvato-User-Agent': self._ANVATO_USER_AGENT,
+                                         'User-Agent': self._ANVATO_USER_AGENT,
+                                     })
+
+        if response.status_code != 200:
+            raise Exception('Error %s in _anvato_get_anvacks.' % response.status_code)
+
+        info = json.loads(response.text)
+        return info
+
+    def _anvato_get_server_time(self, access_key):
+        """ Get the server time from anvato. (not needed)
+        :type access_key: string
+        :rtype dict
+        """
+        url = 'https://tkx.apis.anvato.net/rest/v2/server_time'
+        self._kodi.log('Getting servertime from {url} with access_key {access_key}', url=url, access_key=access_key)
+        response = self._session.get(url,
+                                     params={
+                                         'anvack': access_key,
+                                         'anvtrid': self._generate_random_id(),
+                                     },
+                                     headers={
+                                         'X-Anvato-User-Agent': self._ANVATO_USER_AGENT,
+                                         'User-Agent': self._ANVATO_USER_AGENT,
+                                     })
+        if response.status_code != 200:
+            raise Exception('Error %s.' % response.status_code)
+
+        info = json.loads(response.text)
+        return info
+
+    def _anvato_get_stream_info(self, anvato_info, stream_info):
+        """ Get the stream info from anvato.
+        :type anvato_info: dict
+        :type stream_info: dict
+        :rtype dict
+        """
+        url = 'https://tkx.apis.anvato.net/rest/v2/mcp/video/{video}'.format(**anvato_info)
```

Comment:
Not familiar with this. @basrieter is probably the guy to clarify.

Reply:
This just fills in the `video` placeholder from the `anvato_info` object.

This would be the same as 
```
        url = 'https://tkx.apis.anvato.net/rest/v2/mcp/video/{video}'.format(video=anvato_info['video'])
```

Reply:
Indeed.